inst/extdata/installPackagesDev.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
This SH script isn't actually needed, using the above R script would accomplish the same thing if you replaced the mran URL with the cran url. We may want to go ahead and parameterize line 95 in installPackages.R.